packages/client/src/components/app/Container.svelte

```diff
@@ -25,7 +26,13 @@
   ].join(" ")
 </script>
 
-<div class={classNames} use:styleable={$component.styles} class:wrap>
+<div
+  class={classNames}
+  class:clickable={onClick}
+  use:styleable={$component.styles}
+  class:wrap
+  on:click={onClick}
```

Comment:
If we use `on:click|self={onClick}` here then I think that will sort the bubbling events :+1:

Reply:
Mel pointed out that this prevents clicks on *any* child component from firing - even things like a paragraph for which you would still expect the container on click event to fire for. Let's leave this open for now and I'll jump on at some stage and disable propagating of events from our other components, which is the proper solution to this.

I'd rather get that in before we ship this.